blueocean-analytics-tools/src/main/resources/io/jenkins/blueocean/analyticstools/AnalyticsTools/header.jelly

```diff
@@ -1,32 +1,34 @@
-          ]]>
+                  return normalizedUrl;
+                }
+                var transformer = function(payload) {
+                  payload.data.request.user_ip = '0.0.0.0';
+                  payload.data.request.url = normalizeURL(window.location.pathname);
+                };
+                var _rollbarConfig = {
+                  accessToken: "81f3134dedf44871b9cc0a347b1313df",
+                  captureUncaught: true,
+                  code_version: "${it.blueOceanPluginVersion}",
+                  source_map_enabled: true,
+                  guess_uncaught_frames: true,
+                  transform: transformer
+                };
+                <![CDATA[
```

Comment:
`<![CDATA[` is invalid Javascript. I guess it's just including the text after Jelly processing, normally the whole script block would be wrapped in a `<![CDATA[` tag, but doesn't matter here as long as it doesn't make it to the browser.  e.g. typically I'd see:

```
<script>//<![CDATA[
.. your javascript with & and <
//]]></script>
```